io.ossia.score.yml

```diff
@@ -0,0 +1,333 @@
+id: io.ossia.score
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: ossia-score
+finish-args:
+  # BLE support
+  - --allow=bluetooth
+
+  # Access user's data and media libraries, plug-ins etc
+  - --filesystem=host
+
+  # GPU rendering support
+  - --device=dri
+
+  # Raw input support
+  - --device=input
+
+  # Required for USB & MIDI devices support as well as JACK (/dev/shm) and raw GPIO I/O
+  - --device=all
+
+  # Removable drives support
+  # https://github.com/flatpak/flatpak/issues/2713#issuecomment-1958510347
+  - --filesystem=/media
+  - --filesystem=/run/media
+
+  # X11 is used by default
+  - --socket=x11
+
+  # Wayland still has rendering regressions compared to X11 as of Qt 6.7
+  - --socket=wayland
+
+  # Used by X11
+  - --share=ipc
+
+  # Network access
+  - --share=network
+
+  # Permit PipeWire and user music directory access
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-music
+
+  # PulseAudio is needed for ALSA device access
+  - --socket=pulseaudio
+
+  # Talk to UPower and RTKit services on the session bus
+  - --system-talk-name=org.freedesktop.UPower
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+
+  # Ignore ALSA configuration
+  - --env=ALSA_CONFIG_PATH=
+  - --env=VST_PATH=/app/extensions/Plugins/vst
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+
+  # Required to allow screensaver/idle inhibition
+  - --talk-name=org.freedesktop.ScreenSaver
+
+  # For correct cursor scaling under Wayland
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+
+  # Plugins
+  - --env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2:/app/lib/lv2
+
+  # Config persistence
+  - --persist=ossia
+
+add-extensions:
+  # Linux audio plugins base extension
+  org.freedesktop.LinuxAudio.Plugins:
+     directory: extensions/Plugins
+     version: '23.08'
+     add-ld-path: lib
+     merge-dirs: vst;vst3;lv2
+     subdirectories: true
+     no-autodownload: true
+
+  # Full codec support for FFMPEG
+  org.freedesktop.Platform.openh264:
+    directory: lib/openh264,
+    add-ld-path: .,
+    version: "2.4.1"
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg,
+    add-ld-path: .
+    version: "23.08"
+
+cleanup:
+  - /include
+  - /share/gir-1.0
+  - /share/man
+  - '*.a'
+  - '*.la'
+  - /lib/cmake
+  - /lib/girepository-1.0
+  - /lib/pkgconfig
+  - /lib/*.la
+  - /lib/*.a
+  - /share/pkgconfig
+  - /share/aclocal
+  - /share/gtk-doc
+  - /share/doc
+  - /share/info
+  - /share/man
+  - /man
+modules:
+  # LV2 audio plugin support
+  - shared-modules/linux-audio/lv2.json
+  - shared-modules/linux-audio/lilv.json
+
+  # Audio I/O
+  - name: portaudio
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      - -DBUILD_SHARED_LIBS=OFF
+      - -DPA_ALSA_DYNAMIC=ON
+      - -DPA_USE_JACK=OFF
```

Comment:
why disabling jack when you enable the pipewire socket?

Reply:
ossia already provides direct implementation of JACK through the JACK API (as well as direct implementation of PipeWire audio and midi through the pipewire API) and this causes issues (crashes & deadlocks) when portaudio does its own jack init / teardown in Pa_Initialize / Pa_Terminate.

Reply:
ok